templates/default/nm-gpu-fix.sh.erb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# if nodemanager is running, exit
+systemctl status nodemanager
```

Comment:
As @kouzant  pointed out: `systemctl status might not return and needs user intervention. In my VM that't the behaviour. Use systemctl is-active instead`

Reply:
cc: @jimdowling

Reply:
ok